scripts/test-TLSv1_2-rejected-without-TLSv1_2.py

```diff
@@ -96,6 +99,35 @@ def main():
 
     conversations["sanity"] = conversation
 
+    # sanity check - TLS 1.1 enabled
+    conversation = Connect(host, port)
+    node = conversation
+    ciphers = [CipherSuite.TLS_RSA_WITH_AES_128_CBC_SHA]
```

Comment:
A previous patch in this set uses CipherSuite.tls12Suites. Is there a reason restricting this sanity to a single ciphersuite?

Reply:
1. it's the same cipher that is used in the `sanity` case
2. at the very least, it has to be limited to RSA key exchange ciphers as the test doesn't expect ServerKeyExchange message

so it's easier to have just one cipher here

Reply:
Just for the record, RSA key exchange is not allowed in FIPS mode on RHEL-8.

Reply:
that's something we will have to address for all scripts